### DISCUSSION TITLE
Streaming responses do not appear in chat window until page reload

### DISCUSSION THREAD

**Discussion by squidpickles** [danny-avila/LibreChat#4419] [Troubleshooting]
### What happened?

It doesn't happen 100% of the time, but it happens pretty often. I type a message into a chat box, either in an existing or a new conversation, and hit send. The stop button appears, but nothing else changes on the screen. When the button disappears, the response has completed, but it doesn't appear in the window until I reload the page. If it's a new chat, that also won't show in the chat list until reload. I have tried with both OpenAI and Anthropic models.

I'm building a fresh Docker image each time I start the service using docker-compose, but my installation has been continuously updated for quite some time.

### Steps to Reproduce

Using `git bisect`, I found commit [`473859b0e2cb767f720f8194a749c58af6bbda49`](https://github.com/danny-avila/LibreChat/commit/473859b0e2cb767f720f8194a749c58af6bbda49) that seems to introduce the bug. Not sure whether that's a red herring, but I've tried the commits surrounding it, and some very recent and very old ones, and it does change behavior on that commit.

Seems to be firefox-specific; I don't see it in Safari. No console errors, though.

### What browsers are you seeing the problem on?

Firefox

### Screenshots

https://github.com/user-attachments/assets/147a0a7e-15d2-4f10-b230-af86902cce07

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
I use the app everyday and don’t observe this behavior.

**Reply by squidpickles** [reply]
I do as well (thanks!) and that commit definitely introduced the problem.

But I finally tracked it down, and yeah, it's not actually an introduced bug, but a config error.

I'm using nginx as a reverse proxy, with slightly different configuration for endpoints under `/api/`; that commit moved the health check from `/api/health` to /health`, which broke my config (expecting all API endpoints to be under that `/api/` root).

So thanks, carry on.

**Comment by rakesh-shah** [comment]
@danny-avila 
i have recently noticed this error where after request for a query it keeps on loading and when I reload it , it gives the answer

I have not faced this issue in v0.8.1-rc2 but I am facing the issue in v0.8.2-rc2

can we open this up as a issue?

**Reply by LucaN-22** [reply]
@rakesh-shah 
Have you found a solution for this issue? I encountered it after updating to 0.8.8-rc1
